prover/prover_cli/src/commands/get_proof_progress.rs

```diff
@@ -0,0 +1,56 @@
+use anyhow::Context as _;
+use clap::Args as ClapArgs;
+use sqlx::{postgres::PgPoolOptions, Row};
+use zksync_config::PostgresConfig;
+use zksync_env_config::FromEnv;
+
+#[derive(ClapArgs)]
+pub(crate) struct Args {
+    #[clap(short, long)]
+    l1_batch_number: i32,
+}
+
+pub(crate) async fn run(args: Args) -> anyhow::Result<()> {
+    let pool = PgPoolOptions::new()
+        .max_connections(5)
+        .connect("postgres://postgres:notsecurepassword@localhost/prover_local")
+        .await?;
+
+    let query = sqlx::query(
```

Comment:
Not a strong opinion, but I would use query! as a macro here, I don't see reasons to use a direct method. But feel free to resolve if you disagree.